README.md

```diff
@@ -253,6 +254,43 @@ let myAppConversations = conversations.filter {
 }
 ```
 
+### Decoding a single message
+
+You can decode a single `Envelope` from XMTP using the `decode` method:
+
+```swift
+let conversation = try await client.conversations.newConversation(with: "0x3F11b27F323b62B159D2642964fa27C46C841897")
+
+// Assume this function returns an Envelope that contains a message for the above conversation
+let envelope = getEnvelopeFromXMTP()
```

Comment:
Out of curiosity, can you get an `Envelope` from a `DecodedMessage` or is there another way to do it?

Reply:
I think my understanding was that you'd get an `Envelope` from a push notification potentially, then need to decode it with this? If you've already got a `DecodedMessage` I'm not sure what you'd need to decode at that point.